herald.yaml

```diff
@@ -92,26 +77,49 @@ buckets:
         project_domain_name: "Default"
       container: swift-mirror-test
       region: dc3-a
-    backups:
-      - name: s3-swift-replica
-        backend: minio_s3
-        config:
-          endpoint: "http://localhost:9000"
-          region: local
-          forcePathStyle: true
-          bucket: swift-mirror-test
-          credentials:
-            accessKeyId: minio
-            secretAccessKey: password
-      - name: swift-swift-replica
-        backend: openstack_swift
-        config:
-          auth_url: https://api.pub1.infomaniak.cloud/identity/v3
-          credentials:
-            username: "test:tester"
-            password: "testing"
-            project_name: "PCP-RP63UPV"
-            user_domain_name: Default
-            project_domain_name: Default
-          container: "s3-mirror-test"
-          region: dc3-a
+
+replicas:
+  - name: s3-swift-replica
+    backend: minio_s3
+    config:
+      endpoint: "http://localhost:9000"
+      region: local
+      forcePathStyle: true
+      bucket: swift-mirror-test
+      credentials:
+        accessKeyId: minio
+        secretAccessKey: password
+  - name: swift-swift-replica
+    backend: openstack_swift
+    config:
+      auth_url: https://api.pub1.infomaniak.cloud/identity/v3
+      credentials:
+        username: "test:tester"
+        password: "testing"
+        project_name: "PCP-RP63UPV"
```

Comment:
are the project name fixed when using the dev container or do they change?

Reply:
it doesn't change unless changed manually